scripts/native.proj

```diff
@@ -43,7 +43,7 @@
 
```

Comment:
Suggest renaming this file to `native.amd.proj`

Reply:
OK, and as a result, the reference to it in `appveyor.yml` is updated too. So this renaming of `native.proj` makes `appveyor.yml` becomes exclusively X64 (and thus should be renamed to `appveyor.amd.yml` to be explicitly clear?)

Reply:
There are few other places where `native.proj` is used: https://github.com/gitextensions/gitextensions/search?q=native.proj

> So this renaming of `native.proj` makes `appveyor.yml` become exclusively X64 (and thus should be renamed to `appveyor.amd.yml` to be explicitly clear?)

I don't follow, sorry. We continue to build x86 as well, hence the suggestion to rename to `native.amd.proj` and not `native.amd64.proj`.

Reply:
> There are few other places where `native.proj` is used: https://github.com/gitextensions/gitextensions/search?q=native.proj

OK, due to these reasons, I've decided to revert to the original `native.proj` file name, and have merged the `native.arm64.proj` to it so that the `native.proj` will now support both Intel/AMD and ARM64 builds via the usage of an extra property. And that means the original `appveyor.yml` file won't be changed after all, and such it will continue building the x86/x64 products as originally intended.

I don't have any experience with AppVeyor so I can't do much about how to change `appveyor.yml` to support the extra ARM64 build.

With my latest changes, to build the x86/x64 natives would use the same command as originally intended:
```
dotnet build .\scripts\native.proj
```
while to build the ARM64 natives would require setting an extra `Arm64Build` flag:
```
dotnet build .\scripts\native.proj /p:Arm64Build=true
```

Reply:
You could duplicate the block at line 73-77:
```
- ps: |
    # build .NET
    $cmd = "dotnet build -c Release /p:Arm64Build=true --verbosity q --nologo /bl:.\artifacts\log\build.binlog $buildArgs"
    Invoke-Expression $cmd
    if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode) }
```
Though it will only come into affect after merging the PR.

Reply:
That's a good idea. Few things to note:
- since we're build amd and arm we should distingush between the binlogs, hence this one should be named something like build.arm64.binlog;
- we'll need to distinguish the zips as well, i.e., name the zip something like "GitExtensions-portable.arm64.xyz0123.zip".

Reply:
> You could duplicate the block at line 73-77:

No, it wouldn't work. The command:
```
dotnet build .\scripts\native.proj /p:Arm64Build=true
```
is meant to be executed on a WoA machine (or VM); it won't work if running on x64 platform or VM such as the one used by AppVeyor (I presume.) The reason is that the MSVC compiler for x86/x64 doesn't know how to generate Arm64 binaries.

This is why I suggested a new `appveyor.arm64.yml` to produce the Arm64 GE products, while keeping the existing `appveyor.yml` exclusively for x86/x64 GE products. But `appveyor.arm64.yml` would only work _if_ AppVeyor could run on a WoA platform, and this is a big _IF_ that I'm not sure about.

Reply:
Makes sense.